index.rst

```diff
@@ -10,168 +10,31 @@ as well. This repository and the notebooks are one component of STScI's
 larger `Data Analysis Tools
 Ecosystem <https://jwst-docs.stsci.edu/jwst-post-pipeline-data-analysis>`_.
 
-Installation Instructions
-=========================
+Summary Listing and Descriptions
+================================
+
+The sidebar on the left of this page provides a summery listing of the notebooks,
+and clicking will take you to html-rendered
+versions of the notebooks. These are easily readable, and
+require no special tools beyond your web browser. The notebooks are
+organized by JWST instrument, or are marked as
+cross-instrument. At the top of each notebook you will find a brief
+description of the specific science use case
+that is demonstrated by the notebook, a description of the data that are used,
+and a listing of the data analysis tools that
+are demonstrated by the use case.
 
-Please see the :ref:`installation instructions <install>` for details on how to download,
-install, and run your notebooks.
+The latest versions of the notebooks are contained in the
+`GitHub jdat_notebooks repository <https://github.com/spacetelescope/jdat_notebooks>`_.
 
-Summary of Notebooks
-====================
 
-The table below summarizes the notebooks currently available. The
-Table of Contents on the left of this page will take you to rendered versions
-of the notebooks, which require no special tools beyond your web browser. If you
-do not see an expected notebook listed in the left-hand column, it is not
-currently rendered. The links in the table will take you to the GitHub repository
-location of the notebooks for you to :ref:`download <install>`.
+Installation Instructions
+=========================
 
-.. list-table:: JWST Science Analysis Notebooks
-   :widths: 25 25
-   :header-rows: 1
+To download and execute the notebooks, we recommend that you
+clone this repository to your local computer as described
+in :ref:`installation instructions <install>` for details.
```

Comment:
This line doesn't look like it's rendering correctly.

Reply:
Turns out the file viewer shows internal links this way intentionally(?). The file looks ok outside of github. So this is fine.